woocommerce/woocommerce-ios
# HACK Week: Handle account closing without WPKit

Part 1/3 of the HACK week project to remove WPKit as a dependency.


## Description

This PR adds a method for closing account in `AccountRemote` and updates `AccountStore` to use this method instead of WordPressKit's. 

## Testing instructions

- Log out of the app or skip onboarding if needed.
- On the prologue screen, select Get Started.
- Create a new WPCom account (e.g. using an email address with a "+" postfix).
- After account creation completes, tap the ellipsis button on the empty store picker and select Close Account.
- The account removal request should be successful and you should receive an email about it.

## Screenshots

N/A

---
